src/create-module/command.test.js

```diff
@@ -0,0 +1,51 @@
+const { handler } = require('./command');
+const { createOutputFolder, addProjectConfig } = require('../services/output-project');
+jest.mock("../services/output-project");
+const { logComplete, logError } = require("../services/formatting");
+jest.mock("../services/formatting");
+
+describe('testing handler', () => {
+
+    beforeEach(() => {
+        logComplete.mockReset();
+        logError.mockReset();
+    });
+
+    test('should call logError if createOutputFolder throw Error', async () => {
+        const errorMessage = "errorMessage";
+        const obj = {
+            topic: "randomText1", 
+            difficulty: "randomText2", 
+            projectName: "randomText3"
+        };
+        createOutputFolder.mockImplementation(() => { throw new Error(errorMessage) })
+        await handler(obj);
+        expect(logError).toBeCalledTimes(1);
+        expect(logError).toBeCalledWith(Error(errorMessage));
+    });
+
+    test('should call logError if addProjectConfig throw Error', async () => {
+        const errorMessage = "errorMessage";
+        const obj = {
+            topic: "randomText1", 
+            difficulty: "randomText2", 
+            projectName: "randomText3"
+        };
```

Comment:
Here you can extract `errorMessage` and `obj` outside of every test.

Reply:
thanks, doing it!